### DISCUSSION TITLE
Disable the service account default-editor

### DISCUSSION THREAD

**Discussion by tam-pham-ts** [deployKF/deployKF#173] [Questions & Answers]
How can I disable the service account default-editor when I don't want to use Kubeflow Pipeline APIs?

**Comment by thesuperzapper** [comment]
@tam-pham-ts can you clarify what outcome you are trying to achieve by disabling the `default-editor` service account?

For reference, other components also use the `default-editor` ServiceAccount by default, including Notebooks.

If you are trying to reduce the permissions of the `default-editor` account, I agree this should be possible. We are planning to allow more customization for that in a coming release, but I would love to hear your feedback so we can design that feature.
